### DISCUSSION TITLE
Welcome to shift Discussions!

### DISCUSSION THREAD

**Discussion by KapilDuwadi** [NatLabRockies/shift#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by WajihaNazHasan** [comment]
Hello,

I am planning to make a synthetic grid using NREL / shift. 

I would like to know how the area that is associated with the geometries is calculated. Up to my understanding, it is obtained in sq degrees from the longitudes and latitudes of the polygon and then it is converted to sq km by multiplying it with 6370^2 (6370km is the approx. radius of earth since earth is not round). Can you please tell me is this correct? If not then please justify how are you calculating the area and in which unit?

Also if the area calculated is in km then why is the KW to Peak Consumption Curve is in sq meter and not in sq km? This curve uses the 'sq km area' (as per my understanding) in a 'sq m curve' to get the peak KW using piecewise function.

One last things, i obtained the coordinates from which the area is computed. when i find the actual area from those coordinates using maps and coding it is way different from the one I obtained from SHIFT.
for example:
Area of X, Y coordinates given by shift: 0.85 (not sure of the unit)
Area of X, Y coordinates given by maps: 236m^2 
Area of X, Y coordinates given by a separate code: 296.4m^2

I hope someone is there to clarify my confusion.

Thanks and Best Reards,
